modules/system/ServiceProvider.php

```diff
@@ -304,7 +304,7 @@ protected function registerLogging()
     {
         Event::listen(\Illuminate\Log\Events\MessageLogged::class, function ($event) {
             if (EventLog::useLogging()) {
-                EventLog::add($event->message, $event->level);
+                EventLog::add($event->message, $event->level, $event->context);
```

Comment:
```suggestion
                EventLog::add($event->message, $event->level, ['context' => $event->context]);
```
Good idea @zaxbux! Let's add it under a `context` key though for easier scoping of the data. Keeping in mind as well that we plan on implementing support for the Ignition error page in the core and it provides a lot of contextual information.